drivers/hmis/extensions/grda_warehouse/hud/client_extension.rb

```diff
@@ -12,6 +12,7 @@ module ClientExtension
       has_many :custom_client_addresses, **Hmis::Hud::Base.hmis_relation(:PersonalID, 'CustomClientAddress'), inverse_of: :client
       has_many :hmis_custom_assessments, through: :enrollments
       has_many :hmis_source_custom_assessments, through: :source_enrollments, source: :hmis_custom_assessments
+      has_many :custom_services, through: :source_enrollments
```

Comment:
Elliot's [PR for custom assessments](https://github.com/greenriver/hmis-warehouse/pull/4159/files#diff-736cbdcc0e0094b4fd523aff48ebfb121f27f1e3c09ae5c1e32653760059b561) linked in the PT ticket includes a new controller. Should we also add a new controller for custom services?

Reply:
I don't think we need a controller in this case.